.github/workflows/upload.yml

```diff
@@ -27,6 +27,12 @@ jobs:
         with:
           python-version: '3.x'
       
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+        
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
       - name: Install Python Requirements
         run: pip install -r requirements.txt
 
```

Comment:
These two steps (*Install Python Requirements* and *Test*) can be deleted, or am I mistaken? We already have an entire action dedicated to testing our code.